lib/linter/linter.js

```diff
@@ -91,6 +92,7 @@ const parserSymbol = Symbol.for("eslint.RuleTester.parser");
  * @property {SourceCode|null} lastSourceCode The `SourceCode` instance that the last `verify()` call used.
  * @property {SuppressedLintMessage[]} lastSuppressedMessages The `SuppressedLintMessage[]` instance that the last `verify()` call produced.
  * @property {Map<string, Parser>} parserMap The loaded parsers.
+ * @property {Object} times The times spent on applying a rule to a file (see `stats` option).
```

Comment:
Can we define more specific types instead of using `Object` and `Map` in some of the jsdocs?

Reply:
Of course! 👍  Please see the suggestion below as well as the others from my review.
```suggestion
 * @property {Times} times The times spent on applying a rule to a file (see `stats` option).
```